### PR TITLE
docs: Add badges to README

### LINKED ISSUE #70: docs: add good badges to the readme
### Is your feature request related to a problem? Please describe.
there are number of good looking badges available for the Github Readme some of are given below in the image 
image

here some more badges are available . [Badges](https://github.com/Naereen/badges)
please add the badges shown in the image only and the above link is only for future reference purpose

### Describe the solution you'd like
This badges can be added easily and here is the repo where you can find those in [readme](https://github.com/anitab-org/anitab-org.github.io/blob/develop/README.md) file don't forget to change the repository links in the badges

### Additional context
This issue can be a good first timers issue and as per the context of the [discussion](https://github.com/anitab-org/open-source-programs-backend/issues/69#issuecomment-778649620) I have opened this issue even I can work for this

### PR DESCRIPTION
**Pull request by rmpsc**
### Description

Added badges to the top of README.md to make it more presentable.

Fixes #70 

### Type of Change:

- Documentation

### Checklist:

- [ ] My PR follows the style guidelines of this project
- [ ] I have performed a self-review of my own code or materials

**Code/Quality Assurance Only**
- [ ] My changes generate no new warnings

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,10 @@
-# Open Source Programs (Backend)
 ![Build Status](https://github.com/anitab-org/open-source-programs-backend/workflows/Tests%20Build/badge.svg)
+[![project chat](https://img.shields.io/badge/zulip-join_chat-brightgreen.svg)](https://anitab-org.zulipchat.com/#narrow/stream/235478-anitab-org.2Egithub.2Eio)
```
Comment (codesankalp) [inline README.md]:
```suggestion
[![project chat](https://img.shields.io/badge/zulip-join_chat-brightgreen.svg)](https://anitab-org.zulipchat.com/#narrow/stream/237907-open-source-progs)
```

File: README.md [hunk 2]
```diff
@@ -1,5 +1,10 @@
-# Open Source Programs (Backend)
 ![Build Status](https://github.com/anitab-org/open-source-programs-backend/workflows/Tests%20Build/badge.svg)
```
Comment (codesankalp) [inline README.md:3]:
```suggestion
# Open Source Programs (Backend)

![Build Status](https://github.com/anitab-org/open-source-programs-backend/workflows/Tests%20Build/badge.svg)
```

File: README.md [hunk 3]
```diff
@@ -1,5 +1,10 @@
...
+[![project chat](https://img.shields.io/badge/zulip-join_chat-brightgreen.svg)](https://anitab-org.zulipchat.com/#narrow/stream/235478-anitab-org.2Egithub.2Eio)
+[![License: GPL v3](https://img.shields.io/badge/License-GPL%20v3-blue.svg)](http://www.gnu.org/licenses/gpl-3.0)
+![Open Source Love](https://badges.frapsoft.com/os/v1/open-source.svg?v=102)
+[![Maintenance](https://img.shields.io/badge/Maintained%3F-yes-green.svg)](https://github.com/anitab-org/open-source-programs-backend)
+![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg?style=flat-square)
+# Open Source Programs (Backend)
```
Comment (codesankalp) [inline README.md]:
```suggestion

```

File: README.md [hunk 4]
```diff
@@ -1,5 +1,10 @@
-# Open Source Programs (Backend)
 ![Build Status](https://github.com/anitab-org/open-source-programs-backend/workflows/Tests%20Build/badge.svg)
+[![project chat](https://img.shields.io/badge/zulip-join_chat-brightgreen.svg)](https://anitab-org.zulipchat.com/#narrow/stream/235478-anitab-org.2Egithub.2Eio)
+[![License: GPL v3](https://img.shields.io/badge/License-GPL%20v3-blue.svg)](http://www.gnu.org/licenses/gpl-3.0)
+![Open Source Love](https://badges.frapsoft.com/os/v1/open-source.svg?v=102)
```
Comment (codesankalp) [inline README.md]: It's not visible in GitHub light mode, Can you change this?

**Review by codesankalp — CHANGES_REQUESTED** [review]
Looks good @rmpsc.
I have suggested some changes, Can you please look into it? 
Adding badges below the Project Name is good which I have mentioned in the review,

File: README.md (hunk 2 above)
Comment (rmpsc) [reply README.md:3]: Nice suggestion. I wasn't sure of the preference, thank you.

File: README.md (hunk 4 above)
Comment (rmpsc) [reply README.md]: Can you provide a screenshot please? I can see the badges on my end.

Comment (codesankalp) [reply README.md]: image
@rmpsc Please see the attached image.

Comment (rmpsc) [reply README.md]: @codesankalp Just committed a slightly different badge. Let me know if it resolves the problem.

File: README.md [hunk 5]
```diff
@@ -1,5 +1,11 @@
 # Open Source Programs (Backend)
+
 ![Build Status](https://github.com/anitab-org/open-source-programs-backend/workflows/Tests%20Build/badge.svg)
+[![project chat](https://img.shields.io/badge/zulip-join_chat-brightgreen.svg)](https://anitab-org.zulipchat.com/#narrow/stream/237907-open-source-progs)
+[![License: GPL v3](https://img.shields.io/badge/License-GPL%20v3-blue.svg)](http://www.gnu.org/licenses/gpl-3.0)
+[![Open Source Love](https://badges.frapsoft.com/os/v1/open-source.svg?v=103)](https://github.com/ellerbrock/open-source-badges/)
```
Comment (codesankalp) [inline README.md]:
```suggestion
[![Open Source Love](https://badges.frapsoft.com/os/v1/open-source.png?v=103)](https://github.com/ellerbrock/open-source-badges/)
```
As SVG file is not rendering properly on firefox replace it with png.

**Review by codesankalp — CHANGES_REQUESTED** [review]
Test and Review Done.

Test Results:

1. **On Firefox browser**:
    __Dark theme__:
image
    __Light theme__:
image

2. **On Chrome Browser**:
    __Dark theme__:
image

    __Light theme__:
image

Just a small change for open-source-badge @rmpsc , After this I think your PR will be ready to merge.

File: README.md (hunk 5 above)
Comment (rmpsc) [reply README.md]: @codesankalp Yes, this change looks like it will solve the issue on light theme. Thank you for your help!

**Review by Rahulm2310 — DISMISSED** [review]
LGTM 🎉 .  Good Job @rmpsc 
Changes approved 👍 
image
image

Although, we can add badges for forks and stars as well. @codesankalp What say ?
image

**Comment by codesankalp** [comment]
> Although, we can add badges for forks and stars as well. @codesankalp What say?
> image

As forks and stars counts are already available on the page we can leave it for the future @Rahulm2310

**Review by codesankalp — DISMISSED** [review]
Thanks for your contribution @rmpsc :tada:

File: README.md [hunk 6]
```diff
@@ -1,5 +1,11 @@
 # Open Source Programs (Backend)
+
 ![Build Status](https://github.com/anitab-org/open-source-programs-backend/workflows/Tests%20Build/badge.svg)
+[![project chat](https://img.shields.io/badge/zulip-join_chat-brightgreen.svg)](https://anitab-org.zulipchat.com/#narrow/stream/237907-open-source-progs)
+[![License: GPL v3](https://img.shields.io/badge/License-GPL%20v3-blue.svg)](http://www.gnu.org/licenses/gpl-3.0)
+[![Open Source Love](https://badges.frapsoft.com/os/v1/open-source.png?v=103)](https://github.com/ellerbrock/open-source-badges/)
```
Comment (isabelcosta) [inline README.md]:
```suggestion
[![Open Source Love](https://badges.frapsoft.com/os/v1/open-source.svg?v=103)](https://github.com/ellerbrock/open-source-badges/)
```

**Review by isabelcosta — CHANGES_REQUESTED** [review]
Since most badges are svgs, I suggested you to add other the open source love badge as svg as well, to have good quality :)

Thank you for your contribution @rmpsc 🎉

File: README.md (hunk 6 above)
Comment (rmpsc) [reply README.md]: @isabelcosta The open source badge was svg before, but this svg in specific was having trouble rendering in Firefox. Should I find a different badge or is the png okay?

**Comment by codesankalp** [comment]
@rmpsc You can use this 
```
[![Open Source Love](https://badges.frapsoft.com/os/v3/open-source.svg?v=103)](https://github.com/anitab-org/open-source-programs-backend/)
```
[Open Source Love](https://github.com/anitab-org/open-source-programs-backend/)

Because it's visible on firefox also.

**Comment by rmpsc** [comment]
@codesankalp Just added the badge! All badges are now svg :)

**Review by codesankalp — APPROVED** [review]

**Review by isabelcosta — APPROVED** [review]
Very cool @rmpsc 🙌🏾 
Thank you for your contribution!

**Pull request merged**